catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java

```diff
@@ -157,10 +256,20 @@ private Metacard transformWithExtractors(InputStream input, String id)
             } catch (InvalidPasswordException e) {
                 LOGGER.warn("Cannot transform encrypted pdf", e);
                 return initializeMetacard(id);
+
             }
         }
     }
 
+    private Metacard transformWithoutExtractors(InputStream input, String id) throws IOException {
+        try (PDDocument pdfDocument = inputStreamPDDocumentFunction.apply(input)) {
+            return transformPdf(id, pdfDocument);
+        } catch (InvalidPasswordException e) {
+            LOGGER.warn("Cannot transform encrypted pdf", e);
```

Comment:
There are also two `warn` methods up in `transformeWithExtractors` that should be changed to `debug`.